roles/ngi_reports/tasks/main.yml

```diff
@@ -30,7 +30,7 @@
     site: "upps"
 
 - name: Create Uppsala ngi_reports conf file
-  template: src="ngi_reports_site.conf.j2" dest="{{ ngi_pipeline_conf }}/ngi_reports_{{ site }}.conf"
+  template: src="ngi_reports_{{ site }}.conf.j2" dest="{{ ngi_pipeline_conf }}/ngi_reports_{{ site }}.conf"
```

Comment:
Perhaps we could redo these 2+2 tasks (`template`+`lineinfile`) to use the `with-items` paradigm you've used previously, instead of having `set_fact` two times.